terraform/tfstate.json

```diff
@@ -0,0 +1,818 @@
+{
+  "version": 4,
+  "terraform_version": "1.5.7",
+  "serial": 30,
+  "lineage": "425397de-8394-a003-8a6c-bce854d9cc53",
+  "outputs": {},
+  "resources": [
+    {
+      "mode": "managed",
+      "type": "github_actions_organization_secret",
+      "name": "this",
+      "provider": "provider[\"registry.terraform.io/integrations/github\"]",
+      "instances": []
+    },
+    {
+      "mode": "managed",
+      "type": "github_membership",
+      "name": "this",
+      "provider": "provider[\"registry.terraform.io/integrations/github\"]",
+      "instances": [
+        {
+          "index_key": "Natim",
+          "schema_version": 0,
+          "attributes": {
+            "downgrade_on_destroy": null,
+            "etag": "W/\"68caaa2a1f43f8105fc4f2b11539ab0a4274703153c2003ec2c3a5b4ef9ce6c7\"",
+            "id": "django-commons:Natim",
+            "role": "member",
+            "username": "Natim"
+          },
+          "sensitive_attributes": [],
+          "private": "eyJzY2hlbWFfdmVyc2lvbiI6IjAifQ=="
```

Comment:
@cunla I saw the `private` attribute and it got my attention. I don't know anything about Terraform, so I just wanted to double check -- are these okay to commit to the repo? Not sure if this is a secret or something else.

Reply:
It is just base64 encoded for `{"schema_version":"0"}`.
If nothing vulnerable is configured using terraform, then nothing vulnerable should be stored in the repo.

Reply:
Awesome, thanks!